.gitignore

```diff
@@ -4,5 +4,4 @@ node_modules/
 .idea/
 debug
 .vagrant
-/static/
```

Comment:
IMO, having `static` here is a better choice than putting a `.gitignore` file inside the static directory.

Reply:
👍